brion/plugin/compartmentReportBinary.cpp

```diff
@@ -683,7 +725,9 @@ bool CompartmentReportBinary::_parseMapping()
         _originalGIDs.insert(cell.gid);
     }
 
-    _header.dataBlockOffset = cells[0].dataOffset;
+    // TODO : redundant with _dataOffset = get<uint64_t>(ptr, DATA_INFO +
+    // offset);  Check that.
+    _dataOffset = cells[0].dataOffset;
```

Comment:
Just remove it, I'm sure this one is redundant. My concern at the moment is different.